components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -227,9 +229,10 @@ const ThreadList = ({
                                 disabled={!someUnread}
                                 className={'Button___large Button___icon'}
                                 onClick={handleOpenMarkAllAsReadModal}
+                                marginExtra={true}
                             >
-                                <span className='Icon'>
-                                    <i className='icon-playlist-check'/>
+                                <span className='icon'>
```

Comment:
why the classname change here to `icon`?

Reply:
cause with ### Icon its adding some extra margin by itself, plus we have used ### icon everywhere else

Reply:
> nunnu